wiki/misc/worldtypes.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Adding a Custom World Preset
+---
+
+# Adding custom world presets
+If your mod completely overhauls Minecraft's worldgen, you'll want to create a custom worldtype so you give people the option to use your worldgen or MC's vanilla worldgen.
+Now, what is a custom worldtype you may ask? A custom worldtype is what will show up here: ![createANewWorld.png](createANewWorld.png)
```

Comment:
2 things!
- it's super nice to provide an image here, but it's formatted differently than the other ones. check `get-started.md` in the introduction category to see how to properly add an image
- this explains how to select a world type, not what a world type actually is!

Reply:
"Worldtypes are worldgen options for Minecraft worlds. What they do is change MC's worldgen based on what's defined in a file found in the `minecraft/worldgen/world_preset` directory."

Reply:
how about something like "World types let you change the markup of your Minecraft worlds by configuring how world generation behaves"?

Reply:
> "the markup of your Minecraft worlds"
not entirely sure what "markup" would mean in this context

Reply:
*makeup, sorry, typo'd

Reply:
> "World types are configurable world generation options for Minecraft worlds. They let you change the markup of your Minecraft worlds by configuring how world generation behaves."

how's this